cpp/src/parquet/schema.h

```diff
@@ -416,8 +417,8 @@ class PARQUET_EXPORT ColumnDescriptor {
 // TODO(wesm): this object can be recomputed from a Schema
 class PARQUET_EXPORT SchemaDescriptor {
  public:
-  SchemaDescriptor() {}
-  ~SchemaDescriptor() {}
+  SchemaDescriptor() = default;
+  ~SchemaDescriptor() = default;
```

Comment:
for a C++ newbie, what is the difference here?

Reply:
no differences here